underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -218,6 +225,54 @@ public boolean exists(String path) throws IOException {
     return hdfs.exists(new Path(path));
   }
 
+  @Override
+  public AccessControlList getAcl(String path) throws IOException {
+    FileSystem hdfs = getFs();
+    // TODO(chen): for hadoop <= 2.3, ACL is not supported, there is no getAclStatus method.
+    AclStatus hdfsAcl;
+    try {
+      hdfsAcl = hdfs.getAclStatus(new Path(path));
+    } catch (AclException e) {
+      // When dfs.namenode.acls.enabled is false, getAclStatus throws AclException.
+      return null;
+    }
+    AccessControlList acl = new AccessControlList();
+    acl.setOwningUser(hdfsAcl.getOwner());
+    acl.setOwningGroup(hdfsAcl.getGroup());
+    for (AclEntry entry : hdfsAcl.getEntries()) {
+      // TODO(chen): handle the case where the entry is for default ACL in a directory.
```

Comment:
Can you create JIRA ticket to track each TODO item so we remember to address them before release?

Reply:
Some of the TODOs in this PR can turn into bugs if we don't address it before release. It would be nice to put them in JIRA given that we use JIRA for release blocker tracking.